vllm/sequence.py

```diff
@@ -623,6 +634,8 @@ def __init__(
         computed_block_nums: Optional[List[int]] = None,
         state: Optional[SequenceGroupState] = None,
         multi_modal_data: Optional[MultiModalData] = None,
+        encoder_seq_data: Optional[SequenceData] = None,
+        cross_block_table: Optional[Dict[int, List[int]]] = None,
```

Comment:
If this is just a single block table should the type here be `List[int]` instead of a dictionary?

Reply:
Thanks, committed this change.